cpu/schbench.py

```diff
@@ -40,10 +58,12 @@ def setUp(self):
         '''
         sm = SoftwareManager()
         distro_name = distro.detect().name
+
```

Comment:
any reason for this whitespace? i suggest please drop this

Reply:
done